src/motor-operations.h

```diff
@@ -64,22 +69,30 @@ class MotorOperations {  // Rename SegmentQueue ?
 
   // Wait, until all elements in the ring-buffer are consumed.
   virtual void WaitQueueEmpty() = 0;
+
+  // Get a reference of a FixedArray containing the absolute motors
```

Comment:
so I wanted to use the reference initially, but I got stuck by finding a nice way that makes happy also the tests and the mock/dummy classes that inherits from motor_operations where the method isn't implemented yet and had to return a mock MotorCoordSystem without forcing people to add a mock member variable to return.

> Alternative could as well be to give it a pointer to a MotorCoordSystem variable and have the function-call fill in the values.

I guess I will follow this approach.